features/update/async_accepted/feature.go

```diff
@@ -0,0 +1,134 @@
+package async_accepted
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	enumspb "go.temporal.io/api/enums/v1"
+	"go.temporal.io/api/serviceerror"
+	updatepb "go.temporal.io/api/update/v1"
+	"go.temporal.io/features/features/update/updateutil"
+	"go.temporal.io/features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/sdk/workflow"
+)
+
+type UpdateDisposition int
+
+const (
+	theUpdate       = "theUpdate"
+	theUpdateResult = 123
+	shutdownSignal  = "shutdown_signal"
+
+	succeed       UpdateDisposition = 0
+	failWithError UpdateDisposition = 1
+	failWithPanic UpdateDisposition = 2
+)
+
+var Feature = harness.Feature{
+	Workflows: Workflow,
+	Execute: func(ctx context.Context, runner *harness.Runner) (client.WorkflowRun, error) {
+		if reason := updateutil.CheckServerSupportsAsyncAcceptedUpdate(ctx, runner.Client); reason != "" {
+			return nil, runner.Skip(reason)
+		}
+		run, err := runner.ExecuteDefault(ctx)
+		if err != nil {
+			return nil, err
+		}
+
+		updateCounter := 0
+		requestedSleep := 2 * time.Second
+		sendAnAsyncUpdate := func(
```

Comment:
I usually don't care too much about code organization during tests, but I think people are gonna read these features as how-to's. I think we should just inline this call each time needed. I know it kinda violates reuse, but this is a two-statement helper and arguably the counter is not needed.